Jenkinsfile

```diff
@@ -3701,7 +3701,7 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         model.decoder.bias_activation_fusion=False \
         model.decoder.activations_checkpoint_method='block' \
         model.decoder.activations_checkpoint_num_layers=1 \
-        model.data.data_prefix=[.5,/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document,.5,/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document]"
+        model.data.data_prefix='[/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document]'"
```

Comment:
I don't think we want to change this test. We need to make sure we stay backward compatible.

Reply:
@shanmugamr1992 can you copy-paste the data_prefix you used above here as well? We run the same thing twice to ensure save/restore works.